lib/line/bot/client.rb

```diff
@@ -40,7 +34,9 @@ class Client
       #
       # @param options [Hash]
       # @return [Line::Bot::Client]
-      def initialize(options = {})
+      def initialize(options = { channel_token: ENV["LINE_CHANNEL_TOKEN"],
+                                 channel_id: ENV["LINE_CHANNEL_ID"],
+                                 channel_secret: ENV["LINE_CHANNEL_SECRET"] })
```

Comment:
Even when `Line::Bot::Client.new(channel_id: 'channel_id')`, I think secret and token from env should be respected.